specification/common-types/resource-management/v6/managedidentity.json

```diff
@@ -0,0 +1,114 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "6.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "UserAssignedIdentities": {
+      "title": "User-Assigned Identities",
+      "description": "The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.",
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/UserAssignedIdentity",
+        "x-nullable": true
```

Comment:
Why do we have `"x-nullable": true` in this case?  Every property is nullable on PATCH, are we saying that it is nullable in some other situation besides PATCH?

Reply:
This has been updated now!

Reply:
Could people try to use PUT with nulls in the list, in order to express removing an identity? I guess that makes little sense. Its already obviously not in the list even if you just exclude it from the list, and 'null' doesn't identify which identity you wanted to remove. So yeah, I think I'm good with it not being nullable.